### PR TITLE
Add diagnostic image

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | -- |

## What's in this Pull Request?

Adds diagnostic URL image to auth samples.

### CODE REVIEW

**Review by AlexJerabek — APPROVED** [review]

File: Samples/auth/Office-Add-in-ASPNET-SSO/README.md [hunk 1]
```diff
@@ -100,4 +100,6 @@ Copyright (c) 2019 - 2021 Microsoft Corporation. All rights reserved.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+**Note:** The Index.cshtml file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/auth/Office-Add-in-ASPNET-SSO/README.md:103]: Nit: Colon should not also be formatted per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28750). Here and other instances
```suggestion
**Note**: The Index.cshtml file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good. Just saw a nit.

File: Samples/blazor-add-in/excel-blazor-add-in/excel-blazor-add-in/wwwroot/index.html [hunk 2]
```diff
@@ -23,6 +23,9 @@
 </head>
 
 <body>
+    <!-- The following image URL tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project. -->
+    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/blazor-add-in/outlook-blazor-add-in-run" />
```
Comment (ElizabethSamuel-MSFT) [inline Samples/blazor-add-in/excel-blazor-add-in/excel-blazor-add-in/wwwroot/index.html:27]:
```suggestion
    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/blazor-add-in/excel-blazor-add-in-run" />
```

File: Samples/blazor-add-in/outlook-blazor-add-in/outlook-blazor-add-in/wwwroot/index.html [hunk 3]
```diff
@@ -40,6 +40,9 @@
   </head>
 
   <body>
+    <!-- The following image URL tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project. -->
+    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/blazor-add-in/excel-blazor-add-in-run" />
```
Comment (ElizabethSamuel-MSFT) [inline Samples/blazor-add-in/outlook-blazor-add-in/outlook-blazor-add-in/wwwroot/index.html:44]:
```suggestion
    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/blazor-add-in/outlook-blazor-add-in-run" />
```

File: Samples/hello-world/excel-hello-world/README.md [hunk 4]
```diff
@@ -158,6 +158,6 @@ Copyright (c) 2021 Microsoft Corporation. All rights reserved.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-**Note:** The taskpane.html file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
+**Note**: The taskpane.html file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/hello-world/excel-hello-world/README.md:161]: I'm not seeing the change to the excel-hello-world taskpane.html

Comment (davidchesnut) [reply Samples/hello-world/excel-hello-world/README.md:161]: It's already there from a previous PR. This was just fixing the formatting of the existing note.

Comment (ElizabethSamuel-MSFT) [reply Samples/hello-world/excel-hello-world/README.md:161]: of course, forgot that. Looks good then

**Pull request merged**